capi-configuration/2_gitops-management.md

```diff
@@ -0,0 +1,100 @@
+# Re-using release structures for gitops cluster management
+
+**THIS IS WIP AND MAKES A LOT ASSUMPTIONS CURRENTLY**
+
+## Problem Statement
+
+We want to ...
+1. enable customers to have more agency in upgrading individual components.
+2. give a lower barrier of entry for customers to deeply interact with our product.
+3. allow customers to use known gitops tooling for cluster management.
+
+## Going gitops all the way
+
+A next step in `gitops` adoption would be the addition of having the cluster custom resources in `gitops` as well.
+So the shared git repository would contain not only release information but also the clusters themselves.
```

Comment:
That's interesting. We assumed that customers would setup their gitops repos as they want. Ping @JosephSalisbury

Reply:
Biscuit definitely needs to check that with customers in general, but I tend to agree that limiting that to something controlled by us might sooner or later reach its limits when trying to accomodate for customer peculiarities.

Reply:
yeah, i think providing a structure for customer CRs is gonna be hard, we've already seen a fair bit of differentiation between customers as part of GitOps product discovery